src/mmrelay/meshtastic_utils.py

```
@@ -717,7 +721,10 @@ def connect_meshtastic(passed_config=None, force_connect=False):
     return meshtastic_client
 
 
-def on_lost_meshtastic_connection(interface=None, detection_source="unknown"):
+def on_lost_meshtastic_connection(
+    interface: Any = None,
```

Ruff (0.14.10)

Unused function argument: `interface`

(ARG001)